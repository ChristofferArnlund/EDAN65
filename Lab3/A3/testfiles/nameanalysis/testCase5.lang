int main(int a) {
	return a; // okay: a refers to the parameter
}

int k() {
	int m = a;
}
