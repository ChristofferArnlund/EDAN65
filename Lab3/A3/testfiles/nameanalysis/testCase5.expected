Error at line 6: symbol 'a' has not been declared before this use!
